en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,812 @@
+                        | **Worksheet Name**                 | The name of the sheet with the inventory records.   |
+                        | **Required Cell in A1 Annotation** | **cellName**                                        |
+                        | **Value Of The Cell To Set**       | **newStockValue**                                   |
+                        | **Response Variable Name**         | `respondMessages`                                   |
+                        
+                    4. Click **Save**.
+                    
+                3. To generate a message that confirms that the current inventory record is updated, add a custom message as follows:
+                
+                    1. Click the **+** icon below the last API call you added.                     
+            
+                        ![Notification](../assets/img/tutorials/inventory-service-add-response.png)
+                        
+                    2. Click **Other**.
+                    
+                    3. In the **Statement** field, enter `respondMessages.push("Successfully updated the item: " + itemIDString);`
+                    
+                    4. Click **Save**.
+            
+            6. To specify the action to perform when the `intCellValue > quantity` condition is not met, add a custom statement as follows.
+            
+                1. Click the **+** icon on the **else** path of the `intCellValue > quantity` if statement.
+                
+                    ![Inventory not sufficient](../assets/img/tutorials/inventory-service-else-path-1.png)
+                    
+                2. Click **Other**.
+                
+                3. In the **Statement** field, enter `respondMessages.push("Stock is not enough for the Item: " + itemIDString);`.
+                
+                    Here, you are configuring a message stating that the stock is not enough to be returned if the available stock is less than the quantity requested.
+                    
+                4. Click **Save**.
+                
+        6. To specify the action to perform if the `index > 0` condition is not met, add a custom statement as follows:
+        
+            1. Click the **+** icon on the **else** path of the `index > 0` if statement.
+            
+                ![Inventory not sufficient](../assets/img/tutorials/inventory-service-else-path-2.png)
+                
+            2. Click **Other**.
+            
+            3. In the **Statement** field, enter `respondMessages.push("Cannot find the item: " + itemIDString);`.
+            
+                Here, you are configuring a message stating that the requested item cannot be found when the item name in the order request is not available in the Google sheet.
+                
+            4. Click **Save**.
+               
+16. To send the final response after processing the complete flow of the service, add a respond statement as follows:
```

Comment:
Here we have to say click the `+` icon just before the end icon in the low code diagram.

Reply:
Changed the text and included an image.